app/components/file-menu.js

```diff
@@ -23,6 +23,12 @@ export default Ember.Component.extend({
     share() {
       prompt('Ctrl + C ;-)', window.location.href);
     },
+    embed() {
+      let src = window.location.href.split("?")[0];
+      src += "?numColumns=0";
+      let embedCode = `<iframe width="800" height="600" src="${src}"></iframe>`;
```

Comment:
the resulting embed values are not correctly sanitized, and could pose a XSS risk.

vector:

1. evil twiddler sends cleverly crafted href, ble to escape the src + frame tag in the interpolated string.